app/components/Table.tsx

```diff
@@ -109,41 +110,41 @@ const Table = ({ players, setPlayers }: tableProps) => {
     }, [game?.pots]);
 
     return (
-        <Flex w={'100%'} h={'100%'}>
+        <Flex flex={1}>
             <Flex
                 mx={'auto'}
-                position="relative"
                 justifyContent={'center'}
                 alignItems={'center'}
-                aspectRatio={shouldRotate ? '9 / 12' : '16 / 9'}
+                aspectRatio={shouldRotate ? '9 / 14' : '16 / 9'}
                 backgroundImage={
                     !shouldRotate
                         ? '/table-horizontal.png'
                         : '/table-vertical.png'
                 }
                 backgroundRepeat="no-repeat"
                 backgroundPosition="center"
-                backgroundSize={'contain'}
+                backgroundSize={shouldRotate ? '85%' : 'contain'}
+                maxWidth={'100vw'}
             >
                 <Grid
                     templateAreas={
                         !shouldRotate ? templateGridLarge : templateGridSmall
                     }
                     gridTemplateRows={
-                        !shouldRotate ? 'repeat(4, 1fr)' : 'repeat(6, 1fr)'
+                        !shouldRotate ? 'repeat(4, 1fr)' : 'repeat(7, 1fr)'
                     }
                     gridTemplateColumns={
                         !shouldRotate ? 'repeat(5, 1fr)' : 'repeat(3, 1fr)'
                     }
                     gap={4}
                     h={'100%'}
-                    w={['90%', '100%']}
+                    flex={1}
                     placeItems="center"
                     justifyContent={'center'}
-                    position={'absolute'}
                 >
                     {seatIndices.map((gridIndex: string, index: number) => {
                         const player: Player | null = players[index];
+                        const seatId = index + 1;
```

Comment:
Can yo uexplain why we do this?

Reply:
this is just for readability, since index starts at 0